binding.gyp

```diff
@@ -18,6 +18,9 @@
           'sources': [
               'src/win32-dlfcn.cc'
           ],
+          'include_dirs': [
+            'deps/libffi/windows/<(target_arch)',
```

Comment:
when I was removed that directory,i thought, they could be auto generated.
(so all other system's config directories were gone.
in `deps/libffi/libffi.gyp`, other systems run `configure` before build binaries)

but actually, we still need modify built asm files on the windows system(i hate this...), so i made helper script for this; `deps/libffi/generate-windows.sh`
and then reconsider that files store directory, but that time i was forgot original config dir..
so, binding.gyp set that directory instead `dep/libffi/config/...`

if you want, i can move them to original directory, but currently, this directory only have meaning on the windows system..